server/src/graphql/resolvers/mutations.js

```diff
@@ -15,4 +16,28 @@ module.exports = {
     const user = await User.create({ id: uuidv4(), email, password: hash });
     return user ? true : false;
   },
+
+  upsertWell: async (_, { model }) => {
+    if (!model.id) {
+      model.id = uuidv4();
+    }
+    await Well.upsert({ ...model });
+    return Well.findByPk(model.id);
+  },
+
+  deleteWell: async (_, { id }) => {
+    const deletedWell = await Well.findOne({ where: { id } });
+    await Well.destroy({ where: { id } });
```

Comment:
instead of adding an extra call to the database on line 29 with findOne, you could just return the id of the deletedWell if it succeeds.
<img width="336" alt="Screen Shot 2021-02-02 at 9 04 05 AM" src="https://user-images.githubusercontent.com/31671880/106618889-9c024a80-6535-11eb-9002-fd90fd3b5c63.png">